src/patternfly/components/Brand/brand.scss

```diff
@@ -1,4 +1,17 @@
-// stylelint-disable
+$pf-c-brand--breakpoint-map: build-breakpoint-map();
+
 .pf-c-brand {
+  --pf-c-brand--Width--base: auto;
+  --pf-c-brand--Height--base: auto;
+
+  width: var(--pf-c-brand--Width--base);
+  max-width: 100%;
+  height: var(--pf-c-brand--Height--base);
+
+  @include pf-build-css-variable-stack("--pf-c-brand--Width--base", "--pf-c-brand--Width", $pf-c-brand--breakpoint-map);
+  @include pf-build-css-variable-stack("--pf-c-brand--Height--base", "--pf-c-brand--Height", $pf-c-brand--breakpoint-map);
+
+  &.pf-m-picture {
+    display: flex;
```

Comment:
Could this be `inline-flex`? `<picture>` is inline.

Reply:
Yep